utils/middleware/errorHandler.js

```diff
@@ -1,7 +1,11 @@
-
 const boom = require('@hapi/boom');
+const Sentry = require('@sentry/node');
 const { config } = require('../../config');
 
+Sentry.init({
+  dsn: `https://${config.sentryDns}.ingest.sentry.io/${config.sentryId}`,
```

Comment:
Add environment vars in production environment.

Reply:
Done please check